web/templates/settings/clansettings.html

```diff
@@ -52,13 +53,23 @@
                     placeholder="Icon"
                     value="{{ $icon }}" />
                 </div>
+                <div class="field">
+                  <label class="left aligned">
+                    Clan Status
+                  </label>
+                  <select tabindex="4" name="status">
```

Comment:
```suggestion
                  <select tabindex="4" name="status" value="{{ $status }}">
```

Reply:
i don't think this is correct html but we basically need the existing value to be set as the default